plugins/callback/foreman.py

```diff
@@ -166,16 +166,18 @@ def set_options(self, task_keys=None, var_options=None, direct=None):
         else:
             self._disable_plugin(u'The `requests` python module is not installed')
 
+        self.session = requests.Session()
+        self.session.verify = self._ssl_verify(str(self.get_option('verify_certs')))
```

Comment:
should this also be in the `if https` part?

Reply:
(I have no idea what the default value of verify is…)

Reply:
Default is 1, can be easily changed via config, moved as verify does not make much sense when using http anyway.